### PR TITLE
Centrical - Certification + Remove requirements for custom script

### PR DESCRIPTION
**Pull request by doronneumann**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hi @doronneumann, Thank you so much for your submission. As we ask all our partners to register the connector before we can start the certification process. Please register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration).
Please let me know if you have any questions.
Thank you so much again for working with us.

**Comment by doronneumann** [comment]
Hi @Amjed-Ayoub , I did register the connector and got an approval email.
I sent it from my work email: doron@centrical.com

**Comment by Amjed-Ayoub** [comment]
Hi @doronneumann, Apologies for this mistake. Thank you so much for your submission our Certification Engineers will review your pull request soon. @v-EgorKozhadei FYI.
Thank you so much again for working with us.

**Comment by v-EgorKozhadei** [comment]
Hello @doronneumann, 

I hope you are doing well. 

I am following up with you to see how things are going with updating your connector. 

Please let us know if you have any questions. 

Thank you.

File: certified-connectors/Centrical/Readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Centrical Connector
+
```
[the remark this answers is not in the document]
Comment (doronneumann) [reply certified-connectors/Centrical/Readme.md:2]: Done.

**Comment by doronneumann** [comment]
Hi @v-EgorKozhadei - I fixed all issues that you commented about. Hope everything is OK now.
Thanks.

File: certified-connectors/Centrical/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Centrical",
+    "description": "Centrical dynamically brings together real-time performance management, personalized microlearning, augmented coaching, and advanced gamification to personally guide employees and their managers to success.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Centrical/apiDefinition.swagger.json:5]: @doronneumann could you please change the description just a little bit? It should be concise and reflect the main value and purpose offered by your connector. E.g. `Send learning and performance events to Centrical platform.`
More information about the description can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @doronneumann!
Thank you very much for an update!
Could you please change one more thing before we proceed to the next step?

File: certified-connectors/Centrical/apiDefinition.swagger.json (hunk 2 above)
Comment (doronneumann) [reply certified-connectors/Centrical/apiDefinition.swagger.json]: Sure, changes as you suggested.

**Comment by doronneumann** [comment]
Hi @v-EgorKozhadei  - Fixed description as you suggested.
Best.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @doronneumann, 

I hope you are doing well. 

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions. 

Thank you very much for working with us.

**Pull request merged**